sonar-project.properties

```diff
@@ -0,0 +1,47 @@
+# Organization
+sonar.organization=yseop
+# Must be unique in a given SonarQube instance.
+sonar.projectKey=yseop_vscode-yseopml
+# This is the name and version displayed in the SonarQube UI.
+sonar.projectName=VSCode YseopML Extension
+# sonar.projectVersion=
+
+# Folder containing source code
+sonar.sources=client/src,server/src
+# Encoding of the source code. Default is default system encoding
+sonar.sourceEncoding=UTF-8
+# Excluding tests utilities and files generated by ANTLR from Sonar analysis
+sonar.exclusions=**/src/test/**/*.ts,**/src/grammar/*.ts
+sonar.inclusions=**/src/**/*.ts
+
+##
+## Tests
+##
+# Include all tests files in src folder.
+sonar.tests=client/src/test,server/src/test
+# Load test report generated via jest-sonar-reporter
+sonar.testExecutionReportPaths=target/tests-results/sonar-report.xml
+
+##
+## Coverage
+##
+# Files ignored from coverage analysis
+sonar.coverage.exclusions=**/test/*.test.ts
+
+# Load lcov coverage report
+sonar.typescript.lcov.reportPaths=target/coverage/lcov.info
+
+##
+## Duplications
+##
+# Increase minimum lines for code duplications triggering (default = 10)
```

Comment:
Reason for the increase? 20 seems a bit much.

Reply:
No good reason for the increase. Just a copy-paste.

Reply:
Try to reset (if not done already) and see if it puts us in dire straits or not. The default value – or even something lower xD – would be nice.

Reply:
It was a need to allow specific patterns on AA; default values are sufficient I think.